src/lib/tabs/tab-body.ts

```diff
@@ -53,6 +58,53 @@ export type MdTabBodyPositionState =
 export type MdTabBodyOriginState = 'left' | 'right';
 
 /**
+ * The portal host directive for the contents of the tab.
+ * @docs-private
+ */
+@Directive({
+  selector: '[matTabBodyHost]'
+})
+export class MdTabBodyPortal extends _MdTabBodyPortalBaseClass implements OnInit {
+  /** Whether the tab body should be visible. */
+  visible = false;
+  /** A subscription to events for when the tab body begins centering. */
+  centeringSub: Subscription;
+  /** A subscription to events for when the tab body completes leaving from center. */
+  leavingCenterSub: Subscription;
+
+  constructor(
+    _componentFactoryResolver: ComponentFactoryResolver,
+    _viewContainerRef: ViewContainerRef,
+    private host: MdTabBody) {
```

Comment:
Add underscore to host to show its private

Reply:
Done